docker/README.md

```diff
@@ -7,7 +7,7 @@ This Dockerfile can create images for the following targets, each named after th
 * `dbt-core` _(no db-adapter support)_
 * `dbt-third-party` _(requires additional build-arg)_
 
-For platform-specific images, please refer to that platform's repository (eg. `dbt-labs/dbt-postgres`)
+For platform-specific images, please refer to that platform's repository (eg. `dbt-labs/dbt-adapters/dbt-postgres`)
```

Comment:
Is the intent to point to the README.md for more information or to point to the images themselves? I think it's the former, which should probably just be the root (`dbt-labs/dbt-adapters`).

Reply:
No, I think we should point to a specific repo.  How about this?
```suggestion
For platform-specific images, please refer to that platform's repository (eg. [dbt-postgres](https://github.com/dbt-labs/dbt-adapters/blob/main/dbt-postgres/docker/README.md))
```